pages/agent/v3/linux.md.erb

```diff
@@ -42,6 +42,10 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
 
 The configuration file is located at `~/.buildkite-agent/buildkite-agent.cfg`. See the [configuration documentation](/docs/agent/v3/configuration) for an explanation of each configuration setting.
 
+## Which user the agent runs as
+
+On a generic Linux installation, all commands run as the invoking user.
```

Comment:
I'm not super keen on "invoking user" but can't quite pin down a better alternative.

```suggestion
On Linux installation, all commands run as the logged in user.
```

Reply:
Ah, I see the problem. On a non red hat linux..... ?

Reply:
So, the logged in user might invoke commands as at least two different users, maybe? normal and root/sudo? (my shaky knowledge of Linux may be showing)

Reply:
:+1: